vector/src/main/java/im/vector/riotx/features/rageshake/VectorFileLogger.kt

```diff
@@ -35,13 +35,15 @@ import java.util.logging.Logger
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val LOG_SIZE_BYTES = 20 * 1024 * 1024 // 20MB
-
-private const val LOG_ROTATION_COUNT = 3
+private const val size20MB = 20 * 1024 * 1024
+private const val size50MB = 50 * 1024 * 1024
 
 @Singleton
 class VectorFileLogger @Inject constructor(val context: Context, private val vectorPreferences: VectorPreferences) : Timber.DebugTree() {
 
+    private val maxLogSizeByte = if (vectorPreferences.labAllowedExtendedLogging()) size20MB else size50MB   // 20MB
```

Comment:
Can you join the statements in the if to avoid calling preferences twice

Reply:
Done